tests/RobotFramework/tests/cumulocity/registration/mqtt_representation.robot

```diff
@@ -0,0 +1,27 @@
+** Settings ***
+Resource    ../../../resources/common.resource
+Library    Cumulocity
+Library    ThinEdgeIO
+
+Test Tags    theme:c8y    theme:registration
+Suite Setup    Custom Setup
+Test Teardown    Get Logs
+
+*** Test Cases ***
+
+Mapper republishes entity registration messages with @id
```

Comment:
An entire test suite just for this test looks like an overkill. I'd have just added this test to `registration_lifecycle.robot` itself or even modified some of those existing tests to validate this additional message as well.